docs/content/en/docs/getting-started/production-environment/_index.md

```diff
@@ -54,6 +54,14 @@ Make sure you use single quotes around the values so that your shell does not in
    export EKSA_VSPHERE_PASSWORD='t0p$ecret'
    ```
 
+1. Set License Environment Variable
+
+   If you are creating a licensed cluster, set and export the license variable:
```

Comment:
```suggestion
   If you are creating a licensed cluster, set and export the license variable (see [License cluster]({{< relref "/docs/tasks/cluster/cluster-license" >}}) for an example of a cluster license secret):
```
I'm assuming they need to apply the cluster license secret first. If so, perhaps add a link like above.